projects/observability/src/pages/explorer/explorer-service.ts

```diff
@@ -0,0 +1,20 @@
+import { Injectable } from '@angular/core';
+import { NavigationParams, NavigationParamsType } from '@hypertrace/common';
+import { Filter, toUrlFilterOperator } from '@hypertrace/components';
+import { ScopeQueryParam } from './explorer.component';
+
+@Injectable({ providedIn: 'root' })
+export class ExplorerService {
+  public buildNavParamsWithFilters(scopeQueryParam: ScopeQueryParam, filters: UrlFilter[]): NavigationParams {
```

Comment:
This function is a straight forward one with no dependencies. Can make it a constant function. Or do we want to leave this as a service for possible future usecases? @aaron-steinfeld

Reply:
In general, my rule of thumb is - would it ever make sense to mock out in a test? If so, leave it in a service. Or is it a utility function that does something like capitalization, transformation etc. that is silly to mock? If yes, make it constant.

Here, I think we should be adding dependencies (see below comment) so it would stay in a service.